lib/serial_terminal.pm

```diff
@@ -122,7 +122,7 @@ wait_serial(get_login_message(), 300);
 
 sub get_login_message {
     my $arch = get_required_var("ARCH");
-    return is_sle() ? qr/Welcome to SUSE Linux Enterprise .*\($arch\)/
+    return is_sle() ? qr/(Welcome to SUSE Linux Enterprise .*\(\$arch\)|\blogin:)/
```

Comment:
we cannot relax this for other products, we would miss a bug in any sles in maintenance which miss the package to produce this message. This is specific only for this case in sles 16.
We could check for option in AGAMA_PROFILE_OPTIONS with a regex as we do in other places so we don't mix this regex having one condition on top of is_sle.

Reply:
Changed to use `=~ /software_only_required=true/`.